src/test/resources/graphql/recordprobeoperation.graphql

```diff
@@ -0,0 +1,21 @@
+mutation {
+    recordProbeOperation(request: {
+        operationType: "probify"
+        labware: [{
+            barcode: "STAN-1"
+            probes: [{
+                name: "probe1"
+                lot: "lot1"
+                plex: 1
+            }, {
+                name: "probe2"
+                lot: "lot2"
+                plex: 2
+            }]
+            workNumber: "SGP1"
+        }]
```

Comment:
Also, my understanding from the story description  is that we can associate multiple probe-panels to a labware but there is only one 'lot' and 'plex' fields per labware. Please correct if my understanding is wrong. If so, I need to update the mockup

Reply:
I think that each probe panel used has a lot and a plex number.

Reply:
Ok, Good to know that. I will change Ui